src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileReaderImplTest {
+    private static final String PATH_TO_TEST_INPUT_FILE = "src/test/resources/testInput.csv";
+    private FileReaderImpl fileReader;
+
+    @BeforeEach
+    void setUp() {
+        fileReader = new FileReaderImpl();

```

Comment:
Move the creation of service to the method `@BeforeAll beforeAll()` the service doesn't contain the state and is not changed in different tests.
Fix all similar cases in all test classes where needed.